code/modules/projectiles/pins.dm

```diff
@@ -325,3 +312,18 @@
 
 /obj/item/firing_pin/security_level/pin_auth(mob/living/user)
 	return (only_lethals && !(gun.chambered?.harmful)) || ISINRANGE(GLOB.security_level, min_sec_level, max_sec_level)
+
+// Explorer Firing Pin- Prevents use on station Z-Level, so it's justifiable to give Explorers guns that don't suck.
+/obj/item/firing_pin/explorer
+	name = "outback firing pin"
+	desc = "A firing pin used by the austrailian defense force, retrofit to prevent weapon discharge on the station."
```

Comment:
What was wrong with the away pins?

Reply:
They had the same function of these ones, yet they were a little tasteless by comparison.

Reply:
I meant the name and desc -  that stuff. The "austrailian" lore is kind TG thing, didnt think we would incorporate it is all